examples/covariance/plot_sparse_cov.py

```diff
@@ -54,14 +54,15 @@
 # License: BSD 3 clause
 # Copyright: INRIA
 
+
+
+
```

Comment:
Those extra blank lines are not expected and make the `linter` step fail on the CI

```suggestion
```

You can make sure that your files are automatically formatted by running black on them:

```
pip install black==22.1.0
black examples/covariance/plot_sparse_cov.py
```

Instead of running the second line in the terminal, you can also format the code directly from VS Code using `ctrl-shift-p` + "Format document...".

Reply:
Thank you for the tips, I will correct this.